skycoin/skywire
# Integrate skywire-peering-daemon to mainnet

Fixes #	

 Changes:	
-	Updated `make build` command to build `skywire-peering-daemon` and copy its binary file
-      Added new method `RunDaemon` to Node to enable the automatic startup of `skywire-peering--- 
       daemon` as an external process, and also receive packets via named pipes

How to test this PR:
Clone and place `skywire-peering-daemon` in an adjacent directory to `skywire-mainnet`, run make build & make install. Ensure a skywire-services integration env is set. Run: `make integration-build` & `make integration-run-generic`
